### DISCUSSION TITLE
Help Needed to Get Started with OpenSN

### DISCUSSION THREAD

**Discussion by hityyds** [Open-Sn/opensn#398] [Q&A] [answered]
Hi everyone,

I’m very interested in working with OpenSN, but I’m not sure where to begin. I noticed there’s a folder named tutorials in the repository, but I’m unclear about which files I should start with or the order in which to go through them. Could someone provide some guidance on this?

Additionally, are there any other resources or websites with more detailed and structured tutorials that could help me understand OpenSN better?

Thanks in advance for any advice!

**Comment by andrsd** [comment, marked as answer]
Some useful links:
- [doco pages](https://open-sn.github.io/opensn/index.html)
- [first tutorial](https://open-sn.github.io/opensn/tutorials/lbs/first/first_example.html) - this is the tutorial for the [first_example.lua](https://github.com/Open-Sn/opensn/blob/main/tutorials/lbs/first/first_example.lua)

After you build the code, you can run an input file by:
```./lua/opensn -i <input_file>``` (assuming you are in the build directory)

That should get you started...

**Reply by hityyds** [reply]
Thank you so much for your response!
